src/server/mod.rs

```diff
@@ -284,12 +282,25 @@ impl<State: Send + Sync + 'static> Server<State> {
         self
     }
 
+    // Enable loggin and cookie middleware
+    fn enable_default_middleware(&mut self) {
+        self.middleware(cookies::CookiesMiddleware::new());
+        self.middleware(log::LogMiddleware::new());
+    }
+
+    pub fn into_mock(mut self) -> io::Result<http_service_mock::TestBackend<Self>> {
+        self.enable_default_middleware();
+        http_service_mock::make_server(self)
+    }
```

Comment:
Oh, this code should probably go. We can probably replace this with the new `http_types::Server` trait instead.